tests/sentry/coreapi/tests.py

```diff
@@ -33,9 +33,9 @@ def setUp(self):
         self.pk = self.project.key_set.get_or_create()[0]
         self.helper = self.helper_cls(agent='Awesome Browser', ip_address='198.51.100.0')
 
-    def validate_and_normalize(self, data):
+    def validate_and_normalize(self, data, request_env={}):
```

Comment:
I know this is a test, but be careful with this `{}`. It'd be saver to use `None`.

Reply:
fixed, here and in `normalize()`